package.json

```diff
@@ -6,13 +6,15 @@
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@mui/icons-material": "5.3.1",
+    "@mui/lab": "^5.0.0-alpha.121",
```

Comment:
Looks like there are packages we haven't used. Please don't add if we don't need those.